src/core/account.ts

```diff
@@ -151,10 +151,10 @@ export class Account {
    * Instantiates an account given a private key and a specified account address.
    * This is primarily used to instantiate an `Account` that has had its authentication key rotated.
    *
-   * @param privateKey PrivateKey - private key of the account
-   * @param address The account address
-   * @param args.legacy optional. If set to true, the keypair generated is a Legacy keypair. Defaults
-   * to generating a Unified keypair
+   * @param args.privateKey PrivateKey - the underlying private key for the account
+   * @param args.address AccountAddress - The initial account address before the authentication key was rotated
```

Comment:
why `before`? `aptos.findByPrivateKey` eventually passes the current account (also after key rotation) address on chain

Reply:
Hmm, this is actually wrong. I think it's actually the opposite of this... I will just change this back to `The account address to use`

Reply:
Actually what about this:

`The account address the `Account` will appear as when signing a transaction`